src/data-table/data-table.styles.tsx

```diff
@@ -0,0 +1,156 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import { Checkbox } from "../checkbox";
+import { ArrowDownIcon, ArrowUpIcon } from "@lifesg/react-icons";
+// =============================================================================
+// STYLES
+// =============================================================================
+
+export const TableWrapper = styled.div`
+    width: 100%;
+    border: 2px solid ${Color.Neutral[6]};
+    border-radius: 0.5rem 0.5rem 0 0;
+    overflow: auto;
+`;
+
+export const Table = styled.table<{
+    $addMarginToFirstColumn?: boolean;
+    $addMarginToLastColumn?: boolean;
+}>`
+    width: 100%;
+    border-collapse: collapse;
+    tr,
+    td {
+        padding: 1.5rem 0;
+    }
+    ${(props) => {
+        if (props.$addMarginToFirstColumn) {
+            return css`
+                th:first-child,
+                td:first-child {
+                    padding-left: 1.5rem;
+                }
+            `;
+        }
+    }}
+    ${(props) => {
+        if (props.$addMarginToLastColumn) {
+            return css`
+                th:last-child,
+                td:last-child {
+                    padding-right: 1.5rem;
+                }
+            `;
+        }
+    }}
+`;
+
+export const HeaderRow = styled.tr`
+    background-color: #f5f5f5;
+    height: 5rem;
+    border-bottom: 2px solid ${Color.Neutral[6]};
+`;
+
+export const HeaderCell = styled.th<{ clickable: boolean; maxWidth?: string }>`
+    padding: 1rem;
+    text-align: left;
+    cursor: ${({ clickable }) => (clickable ? "pointer" : "default")};
+    max-width: ${({ maxWidth }) => maxWidth || "auto"};
+    vertical-align: middle;
+    ${TextStyleHelper.getFontFamily("H5", "bold")}
+    color: ${Color.Neutral[1]}
+`;
+
+export const HeaderCellWrapper = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+`;
+
+const getBackgroundColor = (isSelected = false, alternating = false) => {
+    if (isSelected) {
+        return Color.Accent.Light[5];
+    } else if (alternating) {
+        return Color.Neutral[7];
+    } else {
+        return Color.Neutral[8];
+    }
+};
+
+export const BodyRow = styled.tr<{
+    alternating: boolean;
+    isSelected?: boolean;
+    isSelectable?: boolean;
```

Comment:
to add `$` prefix as these are not native html attributes. do update other instances of this